plugins/UsersManager/API.php

```diff
@@ -513,6 +514,22 @@ public function getUsersHavingSuperUserAccess()
     }
 
     /**
+     * Regenerate the token_auth associated with a user.
```

Comment:
Is there any way to grab the authentication and force a re-authentication with the regenerated token (only if user matches the changed one)? Just like the JS side it might be a good thing to not bother people with too many logouts even if regenerating tokens should be a rare action to take.